package.json

```diff
@@ -12,6 +12,13 @@
     "test": "mocha test/*.test.js node_modules/juggler-v3/test.js node_modules/juggler-v4/test.js",
     "posttest": "npm run lint"
   },
+  "files": [
+    "intl",
+    "lib",
+    "docs.json",
+    "index.js",
+    "setup.sh"
```

Comment:
maybe remove the `setup.sh` file, same as https://github.com/strongloop/loopback-connector-cloudant/pull/240#discussion_r391820958

Reply:
We decided to keep `setup.sh` as it's used by `loopback-next/acceptance`. See https://github.com/strongloop/loopback-next/blob/master/acceptance/repository-mysql/setup.sh

Reply:
ah yes.